### PR TITLE
firebase-app-distribution 0.2.0

### PR DESCRIPTION
**Pull request by guness**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by guness** [comment]
icon added and a placeholder token shortened.

File: steps/firebase-app-distribution/0.2.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      You can specify the groups as a comma-separated list:
+
+      `qa-team, trusted-testers`
+    is_required: false
+    summary: null
+    title: Test Groups
+- flags: null
+  opts:
+    description: |
+      Additional flags to be appended to firebase command.
```
[the remark this answers is not in the document]
Comment (guness) [reply steps/firebase-app-distribution/0.2.0/step.yml]: done

File: steps/firebase-app-distribution/0.2.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  release_notes: null
+- opts:
+    description: |
+      The email addresses of the testers you want to invite.
+
+      You can specify the testers as a comma-separated list of email addresses:
+
+      `ali@example.com, bri@example.com, cal@example.com`
+    is_required: false
+    summary: null
```
[the remark this answers is not in the document]
Comment (guness) [reply steps/firebase-app-distribution/0.2.0/step.yml]: done

File: steps/firebase-app-distribution/0.2.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- app: null
+  opts:
+    description: |
+      You can find the App ID in the Firebase console, on the [General Settings page](https://console.firebase.google.com/project/_/settings/general/).
+
+      It is something in the form of:
+
+      `1:1234567890:android:0a1b2c3d4e5f67890`
+    is_required: true
+    summary: null
```
[the remark this answers is not in the document]
Comment (guness) [reply steps/firebase-app-distribution/0.2.0/step.yml]: done

**Comment by richard-bogdan-bitrise** [comment]
@godrei pls merge this PR.

File: steps/firebase-app-distribution/0.2.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,92 @@
+title: '[BETA] Firebase App Distribution'
+summary: |
+  [BETA] Distributes your builds via Firebase App Distribution
+description: |
+  [BETA] Uploads your app binaries to Firebase and sends email to your testers.
+website: https://github.com/guness/bitrise-step-firebase-app-distribution
+source_code_url: https://github.com/guness/bitrise-step-firebase-app-distribution
+support_url: https://github.com/guness/bitrise-step-firebase-app-distribution
```
Comment (godrei) [inline steps/firebase-app-distribution/0.2.0/step.yml]: for the next version pls update to: `https://github.com/guness/bitrise-step-firebase-app-distribution/issues`

File: steps/firebase-app-distribution/0.2.0/step.yml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+published_at: 2019-10-09T14:30:02.127185+03:00
+source:
+  git: https://github.com/guness/bitrise-step-firebase-app-distribution.git
+  commit: 9b48d12ee1f5c2d7898c3af612ee8e95abd07611
+project_type_tags:
```
Comment (godrei) [inline steps/firebase-app-distribution/0.2.0/step.yml]: for the next version pls add `flutter`

**Comment by godrei** [comment]
Hi @guness i posted some possible improvements for your next version. Thanks for the PR!

**Comment by guness** [comment]
> Hi @guness i posted some possible improvements for your next version. Thanks for the PR!

ok, I will prepare next PR by the end of today.

**Pull request merged**